src/components/issue-list-item.component.js

```diff
@@ -56,16 +75,29 @@ export const IssueListItem = ({ type, issue, navigation, language }: Props) =>
       <ListItem
         containerStyle={styles.listItemContainer}
         title={issue.title}
-        subtitle={`#${issue.number} - ${moment(issue.created_at).fromNow()}`}
+        subtitle={
+          issue.state === 'open'
+            ? `#${issue.number} opened ${moment(
```

Comment:
Could you please throw in two translation strings into our locale files and use them here as @lex111 mentioned :)